screens/ChatRoom/index.tsx

```diff
@@ -68,6 +73,8 @@ import {getExploreFeedData} from '../../store/actions/explorefeed';
 import Layout from '../../constants/Layout';
 import EmojiPicker, {EmojiKeyboard} from 'rn-emoji-keyboard';
 import {
+  CHATROOM,
+  DM_FEED,
```

Comment:
why need `DM_FEED`, when user go to DM through GroupChat, then on back press it should back to GroupChat only.